debian/control

```diff
@@ -3,13 +3,12 @@ Section: admin
 Priority: optional
 Maintainer: Ubuntu Developers <ubuntu-devel-discuss@lists.ubuntu.com>
 XSBC-Original-Maintainer: Ryan Harper <ryan.harper@canonical.com>
-Build-Depends: debhelper (>= 9),
+Build-Depends: debhelper-compat (= 13),
```

Comment:
Focal doesn't support compat 13. Do you plan to use this packaging, as is, when building for focal? Or will focal backports continue to use the old packaging?

Reply:
We'll make an ubuntu/focal from the previous commit.